modules/luci-mod-network/htdocs/luci-static/resources/view/network/routes.js

```diff
@@ -183,6 +183,11 @@ return view.extend({
 			o.datatype = 'uinteger';
 			o.placeholder = 10;
 
+			o = s.taboption('advanced', form.Value, 'uidrange', _('User identifier (UID)'));
```

Comment:
I think that the new (undocumented?) option might need a bit more explanation, and also an example of the allowed values (in the help text).